test/integration/granting_permissions_test.rb

```diff
@@ -340,10 +340,10 @@ class GrantingPermissionsTest < ActionDispatch::IntegrationTest
         click_on "Update permissions for MyApp"
         assert_current_url edit_user_application_permissions_path(user, app)
 
-        # the new permissions form exists
+        # the new permissions does not exist
```

Comment:
Absolutely get the need to change this, as the comment is wrong. But it's also not super relevant to the PR. At the same time it is such a minor fix it doesn't seem worth it's own, I'm uncertain...

Reply:
(sorry for the double comment)

Reply:
Yeah, I'm not otherwise changing this file in this PR, but given it's just two line fix of comments, I'm not sure it warrants its own PR. Happy to split it out though if preferred!

Reply:
I've separated that out here: https://github.com/alphagov/signon/pull/2991